Final Code Tweaks (#1)
lucas-tortora/iota.js: documentation/docs/tutorials/value-transactions/02-prepare-the-dev-env.md

:::

:::warning Shimmer Addresses and Outputs

> When it comes to value transactions, the tutorial codebase uses several predefined Shimmer addresses and outputs. If you are going to use other addresses, you will need to update the code accordingly. Likewise, the tutorial codebase assumes a particular state of the addresses and outputs. Before executing the code please ensure your addresses and outputs are in the expected state.

Reviewer: ```suggestion
The tutorial's codebase uses several predefined Shimmer addresses and outputs. If you are going to use other addresses, you will need to update the code accordingly. Likewise, the tutorial codebase assumes a particular state of the addresses and outputs. Before executing the code, please ensure your addresses and outputs are in the expected state.
```
